src/ast/mod.rs

```diff
@@ -1982,12 +1982,20 @@ impl fmt::Display for Statement {
                 Ok(())
             }
             Statement::Truncate {
-                table_name,
+                table_names,
                 partitions,
                 table,
             } => {
                 let table = if *table { "TABLE " } else { "" };
-                write!(f, "TRUNCATE {table}{table_name}")?;
+                write!(
```

Comment:
Perhaps you can use `display_comma_separated` here instead of making a a new `Vec` via join